public/js/updateService.js

```diff
@@ -0,0 +1,40 @@
+document.querySelector("#update").addEventListener("click",event=>{
+    event.preventDefault();
+    const editService = {
```

Comment:
I would probably recommend doing something like this: 
```
//outside of functions at the top of the file as these will never change
const title document.querySelector("#updateTitle");
```

```
//inside the function to check if the value has changed
if (title.value != title.dataset.title) {
    editService.title = title.value;
}
```

This allows us to check if the value has changed from what it was before an edit. This will require the view to be modified to put the original value from the database in as a data-* attribute. We should prepopulate the contents as a value as well to make it easy to edit. Putting the selectors outside and defining them once will make it easier to read and they can be reused across all functions as they will never change. 

Notes on how to use this: 

Data attributes can be accessed like this: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

Note that dashed values are converted to camel case. For example: 
```
<input id="title" data-title-text="This is what the title was">
```
can be accessed by 
```
document.querySelector(#title).dataset.titleText
```

Reply:
This can be done by capturing the values after page load, we can then just compare against them to determine needs to be sent to the update route